### PR TITLE
Update e2e tests in web-frontend 

### PR DESCRIPTION
**Pull request by kael89**
### Issue #:
[No issue] Bringing the e2e tests in web-frontend up-to-date with recent changes

### CODE REVIEW

File: packages/web-frontend/cypress/e2e/dashboardReports.spec.js [hunk 1]
```diff
@@ -50,16 +48,22 @@ describe('Dashboard reports', () => {
...
+      const reportsByGroup = Cypress._.groupBy(reports, 'dashboardGroup');
+
       before(() => {
+        cy.server();
+        cy.route(/\/dashboard/).as('dashboard');
+
         closeDialogsBecauseOfCypressBug();
-        expandDashboardItem(report.name);
+        selectProject(project);
+        cy.wait('@dashboard');
```
Comment (kael89) [inline packages/web-frontend/cypress/e2e/dashboardReports.spec.js:61]: The 'dashboard' related lines is essentially what's changed here

File: packages/web-frontend/cypress/e2e/dashboardReports.spec.js [hunk 2]
```diff
@@ -24,18 +24,16 @@ const closeDialogsBecauseOfCypressBug = () => {
 };
 
 describe('Dashboard reports', () => {
-  const testReportsForProject = (project, reports) => {
-    describe(`Project: ${project}`, () => {
-      const reportsByGroup = Cypress._.groupBy(reports, 'dashboardGroup');
-
+  const testReport = report => {
```
Comment (kael89) [inline packages/web-frontend/cypress/e2e/dashboardReports.spec.js:27]: Only a few lines is new code in this file: the rest are ordering functions to adhere to strict bottom-to-top definitions (functions that get called first are defined last etc)

File: package.json [hunk 3]
```diff
@@ -72,11 +72,13 @@
     "mocha": "^8.1.3",
     "mockdate": "^3.0.2",
     "nodemon": "^2.0.4",
+    "npm-run-all": "^4.1.5",
```
Comment (edmofro) [inline package.json:75]: We already use `concurrently` for building internal dependencies - it might not be necessary to use a different package here?

Comment (edmofro) [reply package.json:75]: Oh I see now, they've come out of psss - all good then

File: packages/web-frontend/cypress/support/actions.js [hunk 4]
```diff
@@ -36,52 +33,33 @@ export const closeEnlargedDialog = () => {
...
 };
 
-/**
- * TODO Remove this method after the bug is fixed
- *
- * @see https://github.com/beyondessential/tupaia-backlog/issues/802
- */
-const waitForProjectDashboardGroupsBecauseOfBug = () => {
-  cy.wait(4000);
-};
```
Comment (edmofro) [inline packages/web-frontend/cypress/support/actions.js:46]: Wahoo!

**Review by edmofro — APPROVED** [review]
All looks good to me!

**Pull request merged**